### ISSUE TITLE
Including Important Scripts into this Repository

### ISSUE THREAD

**Issue by dsand** [so-artn/rts2solib#7]
Hey all --
I am reviewing Dan's detailed notes about this repo from here:
https://docs.google.com/document/d/18Xr6uQQizJHLIEkhqbJ-uOfTRr2v5vcn6HVpIXeIIJ8/edit?usp=sharing

It seems like some important code is on bigartn, but not on this repo.  I would like to resolve this so we can develop away from bigartn.  Unless there is a reason not to do this?  I'm doing to assign Dan and Phil to this.  Dan so that he could chime in if this is a bad way forward.  Let me just copy some important text from that google doc above here:

> Which Python
> When developing these scripts please use the python at 
> #!/home/rts2obs/anaconda3/bin/python3
> 
> Important Scripts and symbolic links:
> /etc/rts2/flat_artn - > /home/rts2obs/git-clones/rts2solib/scripts/flat_artn.py
> Old flat script used in the beginning and end of the night:
>  **/etc/rts2/flat_artn.py** 
> **New flat script used in the beginning and end of the night**

My reading of this is that the true flat script is not on the github repo.  Is that correct?   If so, is there a good reason why?

Barring a good reason, we should try to get this script into the github repo and put the old script into an 'old' directory on the repo.

**Comment by pndaly** [comment]
Yeah, that was me. I persuaded Dan that it was OK. You will see comments (in blue in the original document, by me) 
that say other scripts should do this too. The rationale at the time was that there was a lot of confusion about which
repo was being used and I thought it better to have /etc populated with *all* the correct scripts first before making them
links to the files in the repo. Since we now know which repo we are using, we could probably change this back.

**Comment by dsand** [comment]
Indeed, I see the blue text.  We can discuss, but once it makes sense I think we should have everything on the repo just so we can look at and work on the code off the mountain.

**Comment by dsand** [comment]
OK, Phil Daly and I discussed this tonight, and he agreed to take on this script cleanup job.  This is just a reminder.

**Comment by dsand** [comment]
@pndaly checking in on this.

**Comment by dsand** [comment]
@pndaly checking on this again.

**Comment by pndaly** [comment]
After Dave's / Mike's H-alpha run we found that this caused some minor problems so I took some time out to fix it. The changes I made were:

1. Created /etc/rts2/obsolete;
2. Moved several files from /etc/rts2 to /etc/rts2/obsolete ... interested parties should probably review to check if anything is needed. If not, we can delete this sub-directory (in, say, about a year - just in case!);
3. I added links so that each entry that points to something in /home/rts2obs/git-clones/rts2solib/scripts has both a 'filename' and 'filename.py' soft link to the target ... there is an inconsistency in how files are named in the repo directory so this fixes it (from one point of view);
4. Moved /etc/rts2/guide.py to the repo directory and added soft links to it;
5. I have done nothing with /etc/rts2/samfocustargs.py ... should this be moved to obsolete or the repo directory?
6. I added a /etc/rts2/00CREATE_LINKS.sh to re-create the soft links in case anything needs to be reset.
